elf2cfetbl.c

```diff
@@ -1774,7 +1774,10 @@ int32 GetSectionHeader(int32 SectionIndex, union Elf_Shdr *SectionHeader)
                     SymbolTableDataOffset = SectionHeader->Shdr64.sh_offset + sizeof(Elf64_Sym);
                 }
                 SymbolTableEntrySize = get_sh_entsize(SectionHeader);
-                NumSymbols           = (get_sh_size(SectionHeader) / get_sh_entsize(SectionHeader)) - 1;
+                if (get_sh_entsize(SectionHeader) != 1)
+                {
+                    NumSymbols = (get_sh_size(SectionHeader) / get_sh_entsize(SectionHeader)) - 1;
+                }
```

Comment:
Pseudo-code suggestion: 
SymbolTableEntrySize = get_sh_entsize(SectionHeader);
if (SymbolTableEntrySize == 0)
  NumSymbols = 0;
else
  NumSymbols = (get_sh_size(SectionHeader)/SymbolTableEntrySize) - 1;

This way NumSymbols is always set, fixes the divide by zero check and only calls the subroutine once.